### DISCUSSION TITLE
Conversion of Widget Visibility to Block Visibility

### DISCUSSION THREAD

**Discussion by tnchuntic** [Automattic/jetpack#29170] [Ideas]
Just wanted to ask if there is a plan for conversion of Widget Visibility to Block Visibility? This would be very helpful and useful to have.

I am sure that this will solve a lot of things in the FSE world moving forward.

**Comment by jeherve** [comment]
> Just wanted to ask if there is a plan for conversion of Widget Visibility to Block Visibility?

There is no plan for it at the moment, but I agree that this would be a nice improvement. We'll definitely keep it in mind for future iterations on the feature.

I've created https://github.com/Automattic/jetpack/issues/29176 to keep track of your idea. Whenever we next iterate on widget visibility, we'll start there!

**Comment by jeherve** [comment]
I'll close this discussion now since we have a matching issue to keep track of the idea.
